src/show/components/Navbar/index.js

```diff
@@ -1,63 +1,221 @@
+          className='text-white flex items-center text-2xl cursor-pointer'>
+          <GiTreeGrowth /> raise.me
         </div>
-      </div>
 
-      <div
-        onClick={() => navigate(HOME_PATH)}
-        className='text-white flex items-center text-2xl cursor-pointer'>
-        <GiTreeGrowth /> raise.me
+        <div className='flex flex-row items-center justify-end mx-2 w-1/3'>
+          <div className='flex transition-all duration-300 items-center hover:px-2 hover:py-1'>
+            <Button
+              variant='secondary'
+              iconLeft={<IoConstructOutline className='mx-1' size='1.5rem' />}
+              label='how it works'
+            />
+          </div>
+
+          {isLoggedIn ? (
+            <div className='flex transition-all duration-300 items-center hover:px-2 hover:py-1'>
+              <Button
+                onClick={() => setOpenLogOutModal(true)}
+                variant='secondary'
+                iconLeft={<BsArrowBarLeft className='mx-1' size='1.5rem' />}
+                label='Log out'
+              />
+            </div>
+          ) : (
+            <div className='flex transition-all duration-300 items-center hover:px-2 hover:py-1'>
+              <Button
+                onClick={() => setOpenSignUpModal(true)}
+                variant='secondary'
+                iconLeft={<FiUnlock className='mx-1' size='1.5rem' />}
+                label='sign in'
+              />
+            </div>
+          )}
+
+          <div className='flex border-2 border-primary-50 items-center hover:border-2 hover:rounded-full hover:border-white transition-all duration-300 hover:px-2 hover:py-1'>
+            <Button
+              variant='secondary'
+              iconLeft={<TbStairsUp className='px-0 mx-0' size='1.5rem' />}
+              label='raise it'
+            />
+          </div>
+        </div>
       </div>
 
-      <div className='flex flex-row items-center justify-end mx-2 w-1/3'>
-        <div className='flex transition-all duration-300 items-center hover:px-2 hover:py-1'>
-          <Button
-            variant='secondary'
-            iconLeft={<IoConstructOutline className='mx-1' size='1.5rem' />}
-            label='how it works'
-          />
+      <Modal setVisible={setOpenSignUpModal} visible={openSignUpModal}>
+        <div className='min-h-full flex items-center justify-center py-12 px-6'>
+          <div className='max-w-md w-full space-y-8'>
+            <div className='flex flex-col justify-center items-center'>
+              <BsFillPersonLinesFill size='3rem' className='text-primary-500' />
+              <h2 className='mt-2 text-center text-3xl font-extrabold text-gray-900'>
+                Create a new account
+              </h2>
+            </div>
+            <form className='mt-8 space-y-6'>
+              <div className='rounded-md shadow-sm -space-y-px'>
+                <TextField
+                  required
+                  type='username'
```

Comment:
There's no `username` type, should be text.

Reply:
text acceped